src/Tizen.NUI/src/internal/Application/Application.cs

```diff
@@ -592,6 +636,12 @@ public enum MemoryStatus
         private NUIApplicationMemoryLowEventCallbackDelegate applicationMemoryLowEventCallbackDelegate;
         private LowMemorySignalType memoryLowSignal;
 
+        [UnmanagedFunctionPointer(CallingConvention.Cdecl)]
+        private delegate void NUIApplicationDeviceOrientationChangedEventCallbackDelegate(DeviceOrientationStatus status);
```

Comment:
could you remove `Delegate` word in delegate naming?
https://code.sec.samsung.net/confluence/display/SE/2.9+Delegate%2C+Event

Reply:
I see, I will remove Delegate word in the related names.